opentelemetry-exporter-gcp-logging/src/opentelemetry/exporter/cloud_logging/__init__.py

```diff
@@ -176,7 +188,9 @@ def _set_payload_in_log_entry(log_entry: LogEntry, body: AnyValue):
         else:
             log_entry.text_payload = base64.b64encode(body).decode()
     elif body is not None:
-        log_entry.text_payload = _convert_any_value_to_string(body)
+        log_entry.text_payload = _convert_any_value_to_string(
+            body, "LogRecord.body"
```

Comment:
nit but for `debug_location` i think the more idiomatic thing would be to catch an exception here and log. that way the `code.location` attribute will be correct too